Sources/Plasma/Apps/CMakeLists.txt

```diff
@@ -3,27 +3,18 @@ include_directories("${PLASMA_SOURCE_ROOT}/FeatureLib")
 include_directories("${PLASMA_SOURCE_ROOT}/NucleusLib")
 include_directories("${PLASMA_SOURCE_ROOT}/PubUtilLib")
 
-if(WIN32)
-    if(PLASMA_BUILD_CLIENT)
-        add_subdirectory(plClient)
-        add_subdirectory(plCrashHandler)
-    endif()
+if(PLASMA_BUILD_CLIENT)
+    add_subdirectory(plClient)
 
-    if(PLASMA_BUILD_LAUNCHER)
-        add_subdirectory(plUruLauncher)
+    if(WIN32)
+        add_subdirectory(plCrashHandler)
     endif()
 endif()
 
-if(APPLE)
-    if(PLASMA_BUILD_CLIENT)
-        add_subdirectory(plClient)
-    endif()
+if(PLASMA_BUILD_LAUNCHER AND WIN32)
+    add_subdirectory(plUruLauncher)
 endif()
 
 if(PLASMA_BUILD_TOOLS)
-    add_subdirectory(plPythonPack)
-    add_subdirectory(plFileSecure)
-    add_subdirectory(plFileEncrypt)
-    add_subdirectory(plPageInfo)
-    add_subdirectory(plPageOptimizer)
+    add_subdirectory(SoundDecompress)
```

Comment:
This is now the only thing outside Sources/Tools that's conditional on `PLASMA_BUILD_TOOLS` - maybe it should use a different option? I don't really have any better ideas though. We probably don't want to build it unconditionally, but does it need its own `PLASMA_BUILD_SOUNDDECOMPRESS` option? I guess we could group it under `PLASMA_BUILD_LAUNCHER`..?

Reply:
I'm definitely open to suggestions here, but don't really have any better ideas myself 😕